micro-s3/build.gradle

```diff
@@ -5,6 +5,10 @@ dependencies {
 	compile 'com.amazonaws:aws-java-sdk:' + s3Version
 	compile project(':micro-core')
 	compile project(':micro-manifest-comparator')
+    compile group: 'org.springframework', name: 'spring-test', version: '4.0.5.RELEASE'
```

Comment:
these should probably be testCompile by the looks of things.
Also there is a top level gradle.properties file where dependency versions can be specified. Keeps them the same across all included plugins.